<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>LoadingWindow</class>
  <widget class="QDialog" name="LoadingWindow">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>400</width>
        <height>250</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Loading</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
        <widget class="QLabel" name="status_label_1">
          <property name="text">
            <string>Measuring Finds: 0/0</string>
          </property>
          <property name="font">
            <font>
              <pointsize>12</pointsize>
            </font>
          </property>
        </widget>
      </item>
      <item row="1" column="0">
        <widget class="QProgressBar" name="progress_bar_1">
          <property name="value">
            <number>0</number>
          </property>
        </widget>
      </item>
      <item row="2" column="0">
        <widget class="QLabel" name="status_label_2">
          <property name="text">
            <string>Measuring Models: 0/0</string>
          </property>
          <property name="font">
            <font>
              <pointsize>12</pointsize>
            </font>
          </property>
        </widget>
      </item>
      <item row="3" column="0">
        <widget class="QProgressBar" name="progress_bar_2">
          <property name="value">
            <number>0</number>
          </property>
        </widget>
      </item>
      <item row="4" column="0">
        <widget class="QLabel" name="status_label_3">
          <property name="text">
            <string>Computing Similarity Matrix...</string>
          </property>
          <property name="font">
            <font>
              <pointsize>12</pointsize>
            </font>
          </property>
          <property name="visible">
            <bool>false</bool>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources/>
  <connections/>
</ui>